src/Compilers/VisualBasic/Test/Symbol/DocumentationComments/DocCommentTests.vb

```diff
@@ -12197,7 +12197,9 @@ xmlDoc)
                         xmlDoc = CStr(stringMapper(xmlDoc))
                     End If
 
-                    Assert.Equal(expectedDocXml.Value.Replace(vbLf, vbCrLf).Trim(), xmlDoc)
+                    Assert.Equal(
+                        expectedDocXml.Value.Replace(vbLf, vbCrLf).Trim(),
+                        xmlDoc.Replace(vbLf, vbCrLf).Trim())
```

Comment:
This breaks the windows tests. It replaces \r\n with \r\r\n 

You could replace vbCrLf -> vbLf -> vbCrLf to make it work for both?

Reply:
Ah yes. Will fix that.

Reply:
Actually even simpler idea here. The `expectedDocXml` is guaranteed to have `vbLf` newlines as VB XML literals strictly follow the XML specification. I can just translate `xmlDoc.Replace(vbCrlf, vbLf)` and that should be sufficient.